src/validators/citizenship-multiple.js

```diff
@@ -1,53 +1,145 @@
+import store from 'services/store'
```

Comment:
I normally don't like to import the store directly, as this can be hard to test using mock stores. Ideally, the validation function should be called with both `data, formType` params. However it became clear to me that the validator *classes* are instantiated in many areas of the code, and modifying all of those places to include the current `formType` value felt too risky to try right away. I think we could eventually change to that, after cleaning up some of the places that use validator functions (such as navigation).